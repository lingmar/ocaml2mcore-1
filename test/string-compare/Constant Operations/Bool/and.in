let bool_and b1 b2 =
  b1 && b2
in 42

