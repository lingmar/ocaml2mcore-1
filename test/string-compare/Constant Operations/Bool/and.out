mexpr
let bool_and = lam b1. lam b2.
  if b1 then b2 else false
in 42
